.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 max-line-length=120
-exclude = .git,__pycache__,dist
\ No newline at end of file
+exclude = .git,__pycache__,specification-release/apigee_client.py,specification-release/spec_uploader.py,dist
```

Comment:
If this is using deploy-service then you'll likely not need this any more

Reply:
changed